rust/tests/write_exploration.rs

```diff
@@ -0,0 +1,462 @@
+extern crate chrono;
```

Comment:
is this file for demo purpose only? or do you expect it to be merged as part of the patch set?

Reply:
i was planning to merge it as a guidepost for future iterations of write support, but I'm happy to remove it if you think that is better.

Reply:
Excellent. I personally think it is good and fine to run this test in CI.